README.md

```diff
@@ -54,7 +54,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:6.7.0-SNAPSHOT'
```

Comment:
This should remain on 6.7 until it becomes stable.

Reply:
Are you sure about it? Because after I create a tag with name `v6.7.0-beta.1` [this function](https://github.com/mapbox/mapbox-java/blob/80424defcbeceb8505ce2876212ea146829d33f1/gradle/artifact-settings.gradle#L35) returns `6.8.0-SNAPSHOT`.

Reply:
That's a mistake then, we're still working towards 6.7, until it's stable. We can merge as is and cut a ticket to resolve though.

Reply:
#1467